wire-ios-data-model/Source/MLS/MLSUserID.swift

```diff
@@ -48,18 +48,8 @@ public struct MLSUserID {
         )
     }
 
-    public init?(
-        userID: String,
-        domain: String
-    ) {
-        guard
-            userID.isNonEmpty,
-            domain.isNonEmpty
-        else {
-            return nil
-        }
-
+    public init?(userID: String, domain: String) {
+        guard !userID.isEmpty, !domain.isEmpty else { return nil }
```

Comment:
Maybe that's easier to read?
```suggestion
        if userID.isEmpty || domain.isEmpty { return nil }
```